syntax = "proto3";

message ReadMessage {
	string messageId = 1;
}